scripts/settings.yaml

```diff
@@ -0,0 +1,511 @@
+---
+- group: base_station_mode
+  name: enable
+  type: boolean
+  units:
+  default value: 'false'
+  enumerated possible values: true,false
+  Description: Operate in base station mode
+  Notes: 'This flag ultimately determines whether the SBP message with identifier
+    MSG_BASE_POS will be calculated and sent.  Logically, setting this attribute to
+    "true" sets the Local Piksi as a base station and configures the unit to send
+    its surveyed location coordinates to the other Piksi(s) with which the base
+    station is communicating.  If "true", the Piksi that receives the base station''s
```

Comment:
The last sentence makes it sound to me like you have to have base station mode set on the rover in order to generate pseudo absolute positions.